v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-tcn/auto-ml-forecasting-demand-tcn.ipynb

```diff
@@ -0,0 +1,1470 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Model name: {model_name_str}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.3.3 Build pipeline\n",
+    "\n",
+    "Next, we build the pipeline and kick off a run which will select the best TCN model and register it in the workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_pipeline = Pipeline(\n",
+    "    description=\"training_pipeline_tcn\",\n",
+    "    workspace=ws,\n",
+    "    steps=[automl_step, register_model_step],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Is the TCN model trained? {IS_TCN_MODEL_TRAINED}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "if not IS_TCN_MODEL_TRAINED:\n",
+    "    print(\"Training new AutoML model ...\\n---\")\n",
+    "    training_pipelin_run = experiment.submit(training_pipeline)\n",
+    "    training_pipelin_run.wait_for_completion(show_output=False)\n",
+    "    IS_TCN_MODEL_TRAINED = True\n",
+    "else:\n",
+    "    from azureml.train.automl.run import AutoMLRun\n",
+    "    from azureml.pipeline.core.run import PipelineRun\n",
+    "\n",
+    "    PIPELINE_RUN_ID = (\n",
+    "        \"<Replace with the PipelineRunId used for training the best TCN model>\"\n",
+    "    )\n",
+    "    training_pipelin_run = PipelineRun(experiment=experiment, run_id=PIPELINE_RUN_ID)\n",
+    "    print(f\"Using previously trained model. Pipeline run ID: {PIPELINE_RUN_ID}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Train the baseline model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use Naive model as our baseline. To train it, we kick off another automl experiment with the following settings. Please note that we disabled DNN (`enable_dnn=False`) and added the Naive model to the `allowed_models` list. We also set the number of iterations to 1 since we only interested in training one specific model which is Naive."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl_config = AutoMLConfig(\n",
+    "    task=\"forecasting\",\n",
+    "    debug_log=\"baseline.log\",\n",
+    "    primary_metric=\"normalized_root_mean_squared_error\",\n",
+    "    experiment_timeout_hours=1,\n",
+    "    iterations=1,\n",
+    "    training_data=train_dataset,\n",
+    "    label_column_name=target_column_name,\n",
+    "    compute_target=compute_target,\n",
+    "    enable_early_stopping=True,\n",
+    "    n_cross_validations=2,\n",
```

Comment:
Shall we make a note that we're not actually using cross-validations in TCN?

Reply:
Good point. I added a few sentences on the difference between configs for the baseline and TCN models.